lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
@brynary had a good suggestion to use `Pathname` for this, just to avoid messing up any of the fiddly bits in dealing with path names. I thought there was a `relative_to?`-like method on it already, but it doesn't look like it. There might be a way to use [`#relative_path_from`](http://ruby-doc.org/stdlib-2.2.0/libdoc/pathname/rdoc/Pathname.html#method-i-relative_path_from) though?